build.gradle.kts

```diff
@@ -66,10 +66,17 @@ val previousVersion: Version by lazy {
         versionObj
 }
 
-val isNewVersion = previousVersion != versionObj
+val ossrhConfigured = getProjectProperty("ossrhUser") != null
+val canSign = getProjectProperty("signing.keyId") != null
```

Comment:
Shouldn't this be `signingKeyId` given it is defined as such in the environment vars of GitHub Action? Or does it get this info from somewhere else?

Reply:
Yes, I forgot to change the key logic for github actions. Thanks for pointing this out!